Formula/pbc-sig.rb

```diff
@@ -21,10 +21,19 @@ class PbcSig < Formula
 
   depends_on "pbc"
 
+  # Fix -flat_namespace being used on Big Sur and later.
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
+    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
+  end
+
   # https://groups.google.com/forum/#!topic/pbc-devel/ZmFCHZmrhcw
   patch :DATA
 
   def install
+    # Disable -fnested-functions CFLAG on ARM, which will cause it to fail with:
+    # incompatible redeclaration of library function 'pow'
+    inreplace "configure.ac", "-fnested-functions", "" if Hardware::CPU.arm?
```

Comment:
This will need to be done in `configure` to do anything as build isn't regenerating it.

Reply:
Whoops, I'll fix that.